vue.json

```diff
@@ -0,0 +1,57 @@
+{
+  "extends": ["plugin:vue/vue3-essential"],
+  "parserOptions": {
+    "extraFileExtensions": [".vue"]
+  },
+  "plugins": ["vue"],
+  "rules": {
+    "vue/attributes-order": "error",
+    "vue/component-tags-order": "error",
+    "vue/no-lone-template": "error",
+    "vue/no-multiple-slot-args": "error",
+    "vue/no-v-html": "error",
+    "vue/order-in-components": "error",
+    "vue/this-in-template": "error",
+    "vue/attribute-hyphenation": "error",
+    "vue/component-definition-name-casing": "error",
+    "vue/first-attribute-linebreak": "error",
+    "vue/html-closing-bracket-spacing": "error",
+    "vue/html-end-tags": "error",
+    "vue/multiline-html-element-content-newline": "error",
+    "vue/mustache-interpolation-spacing": "error",
+    "vue/no-multi-spaces": "error",
+    "vue/no-spaces-around-equal-signs-in-attribute": "error",
+    "vue/no-template-shadow": "error",
+    "vue/one-component-per-file": "error",
+    "vue/prop-name-casing": "error",
+    "vue/require-default-prop": "error",
+    "vue/require-explicit-emits": "error",
+    "vue/require-prop-types": "error",
+    "vue/v-bind-style": "error",
+    "vue/v-on-event-hyphenation": "error",
+    "vue/v-on-style": "error",
+    "vue/v-slot-style": "error",
+    "vue/html-quotes": "error",
+
+    "vue/html-self-closing": [
+      "error",
+      {
+        "html": {
+          "void": "always"
+        }
+      }
+    ],
+
+    "putout/putout": "off",
+    "max-len": "off",
```

Comment:
`max-len` is set to 120 chars in `prettier.json`. I set a higher limit specifically for Prettier, and my intention was to still warn about lines longer than 120 even when using Prettier, cuz 120 is too long. Do most of Vue projects have lines longer than 120 chars? Is it markup or something? If that's so, I would still enable `max-len`, but with a higher limit (like 140 maybe, 160 tops).

Reply:
I had to disable this rule because it was conflicting with something. There is no specific need for lines longer than 120 chars. I'll see what else I can do so max-len can be enabled again.